data/hai missions.txt

```diff
@@ -197,6 +200,7 @@ mission "Unfettered Jump Drive 1"
 	source
 		attributes "unfettered"
 	on offer
+		log "Sold a Jump Drive to the Unfettered for much more than it's worth. There's clearly an opportunity for profit here."
```

Comment:
Jump drive should be lower cased.
I'd change "there's" to "there is," but that's preference.